### PR TITLE
Improves SEC-450: Url of the "Order" buttons should be translated

### PR DESCRIPTION
**Pull request by Screenfeed**
- Use a new API route, with a new data format.
- All data in each plan is available in different language.
- Monthly an yearly prices now have their own checkout URL.
- Prices, currency, and currency position can have different values depending on the language: we'll be able to display prices in € for French people.
- Today we display a "20% OFF" discount: we'll be able to display a "2 months free" if we want.

### CODE REVIEW

File: inc/admin/upgrader.php [hunk 1]
```diff
@@ -296,4 +296,10 @@ function secupress_new_upgrade( $secupress_version, $actual_version ) {
 			}
 		}
 	}
+
+	// < 1.2.6.1
```
Comment (geoffreycrofte) [inline inc/admin/upgrader.php:300]: PHPCS: Inline comment should begin with an uppercase and end with a `.`
:p

**Review by geoffreycrofte — APPROVED** [review]
Seems good :)

File: inc/admin/upgrader.php (hunk 1 above)
Comment (Screenfeed) [reply inc/admin/upgrader.php:300]: You have this? Since the comment starts with a `<`, you shouldn't have it (I don't).

Comment (geoffreycrofte) [reply inc/admin/upgrader.php:300]: I was not sure for the `<`. No I didn't run `phpcs`. Just trolling ;p

Comment (Screenfeed) [reply inc/admin/upgrader.php:300]: Nice try ^^

**Pull request merged**